1400-continuous-integration.md

```diff
@@ -5,8 +5,35 @@ aliases: [ "05-continuous-integration" ]
 
 ## With GitHub Actions 
 
-You tipi subscription can be used to run your build and test your project in GitHub actions.
+The tipi subscription can be used to run your build and test your project in GitHub actions.
 
 Run `tipi ci` to generate the YAML configuration for GitHub CI.
 
-See [Command Line authentication](04-authentication#authentication-in-continuous-integration-context) to handle authentication in CI workflows.
+### Authenticating on tipi.build from CI workflows
+Information are available as reference here : [Command Line authentication](13-environment-variables#command-line-authentication) to handle authentication in CI workflows.
+
+An example `.github/workflows/ci.yaml` would look like the following : 
+
+```yaml
+name: build 
+# This workflow is triggered on pushes to the repository.
+on: [push]
+
+env: 
+  TIPI_ACCESS_TOKEN: "${{ secrets.TIPI_ACCESS_TOKEN }}"
+  TIPI_REFRESH_TOKEN: "${{ secrets.TIPI_REFRESH_TOKEN }}"
+  TIPI_VAULT_PASSPHRASE: ${{ secrets.TIPI_VAULT_PASSPHRASE }}
```

Comment:
We should link to github's Actions documentation regarding CI secrets: 
https://docs.github.com/de/actions/security-guides/using-secrets-in-github-actions